libs/dh/shared/feature-authorization/src/lib/scope-storage.service.ts

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
nitpick: rename the file to `scope-storage.ts` since it's just a simple class and not an Angular service.

Reply:
fixed